### ISSUE TITLE
Only half of image is drawn or not draw some images

### ISSUE THREAD

**Issue by abinhho** [niklasvh/html2canvas#2278]
# Please follow the general troubleshooting steps first:

- [x] You are using the latest [version](https://github.com/niklasvh/html2canvas/releases)
- [x] You are testing using the non-minified version of html2canvas and checked any potential issues reported in the console

### Bug reports:

I am using html2canvas to capture entire of webpage but got problem some image can not draw or draw half of image. See my attachments.

This is my code
```
var config = {
  proxy: base_url + "proxy",
  logging: true
};

html2canvas(document.body, config).then(function (canvas) {
  my_image = canvas.toDataURL("image/jpeg");

  resolve(true);
});
```
I tried html2canvas 1.0.0-rc.1 and html2canvas 1.0.0-rc.5 but not luck. Also try set image timeout option, also try to remove some html element to make sure html page not too large but same result.

#### This is origin image

![alt text](https://i.stack.imgur.com/jMPKs.png)

####  This is the capture result (please don't mind the line in image, I just added it) enter image description here
![alt text](https://i.stack.imgur.com/yqZFy.jpg)

Any help!
### Specifications:

 * html2canvas version tested with: `1.0.0-rc.1`, `1.0.0-rc.5`
 * Browser & version: `Chrome 60`
 * Operating system: `MACOS`

**Comment by xbw187** [comment]
Hello, has your problem been solved?

**Comment by abinhho** [comment]
Yes, I found problems is max canvas height around 16.000px, my canvas went to 30.000px height.
So I split to many parts and store it.

**Comment by yyxhqm** [comment]
你好，请问该问题该怎么解决？因为我也是由于要截取的内容太大，导致最后一部分内容缺失

**Comment by xbw187** [comment]
What is the resolution of your images？
&nbsp;


蓝猫玩
xbw187@foxmail.com



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "袁永潇"<notifications@github.com&gt;; 
发送时间: 2020年8月15日(星期六) 下午3:43
收件人: "niklasvh/html2canvas"<html2canvas@noreply.github.com&gt;; 
抄送: "薛定谔的猫"<xbw187@qq.com&gt;; "Comment"<comment@noreply.github.com&gt;; 
主题: Re: [niklasvh/html2canvas] Only half of image is drawn or not draw some images (#2278)





 
你好，请问该问题该怎么解决？因为我也是由于要截取的内容太大，导致最后一部分内容缺失

**Comment by abinhho** [comment]
I don't think image resolution was problem here, if have you can split canvas to many many parts. Sure it works.

**Comment by zoubeihua** [comment]
我的页面高度是7w像素 底部一些页面图片截取是白色的，怎么解决呢
